src/isSymbol.js

```diff
@@ -0,0 +1,32 @@
+import { type } from 'ramda';
+import curry1 from 'ramda/src/internal/_curry1';
+
+/**
+ * Checks if input value is a Symbol.
+ *
+ * @func isSymbol
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol|MDN Symbol}
+ * @example
+ *
+ * RA.isSymbol(Symbol('1')); //=> true
+ * RA.isSymbol(Symbol(1)); //=> true
+ * RA.isSymbol('string'); //=> false
+ * RA.isSymbol(undefined); //=> false
+ * RA.isSymbol(null); //=> false
+ */
+const isSymbol = curry1(val => {
+  return (
+    !!val &&
+    (typeof val === 'symbol' ||
+      (typeof val === 'object' && type(val) === 'Symbol') ||
+      (typeof val === 'object' || val.constructor.name === 'Symbol'))
```

Comment:
I have added some checks on top of the suggested in the issue, for example, `!!val` and checking constructor name since some polyfills might define `Symbol` as objects. What do you think?

Reply:
I'd remove `!!val`, we are concerned here by happy path only, we don't need to eliminate negatives.

Reply:
Also having check for constructor name is  a **no-go**. Everything with the constructor name symbol will be matched  by this function and we don't want that. In modern javascript there is [toStringTag](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) symbol which solves this problem.

Reply:
I understand. I have seen the usage of `toStringTag` in some articles, that is indeed correct. I should have checked the details on `type` before!

Reply:
Done dc6d287c652e63f8ac31c3511ac97eb238bcf7a4 😄